### ISSUE TITLE
How to label our nodes

### ISSUE THREAD

**Issue by illip** [chin-rcip/chin-rcip#42]
Currently in the Target Model, we don't document the labeling of the nodes (`E1->rdfs:label->rdfs:Literal`). In fact we mention: 

> Labels can be attached to any resource across the target model and, since they can be so easily added, they will not be displayed in the schemas below to facilitate readability.

However, I think we have to define a labeling protocol for each type of node that we describe. For the `E21_Person` node, perhaps it's enough to state the full appellation of the person. However, how do we name a `E52_Time-Span`?

At the same time, we should document if this labeling would be automatically generated by using other values in the dataset.

**Comment by stephenhart8** [comment]
We need to automatically generate those labels from partitioning values within the dataset. For example, the Birth event of the person could be the "Birth event of" + [Full Appellation]. But that would require to check all the nodes of our model.
We also need to have in mind that this would need to be bilingual.

The question of the labels also concerns the vocabularies, but that would probably be the topic of another issue...

**Comment by Habennin** [comment]
Yes, it would be a very good idea to specify a common labelling practice. In the sample mapping I generated for you, I put in some indications of how that can go. This step is often neglected in favour of the #43 discussion (because that issue is crucial for connectivity), but is also very important since humans interact with the labels of nodes and not their IRIs. 

Once you land on your overall URI/IRI policy, you might connect it to your label policy for the same kinds of nodes

I assume you will specify you URI/IRI policy relative to classes in the ontology, so you could specify your labelling policy in the same way.

It may be interesting to chat with SARI about their IRI pattern because they work in the same direction.
